src/routes/users.route.js

```diff
@@ -1,26 +1,24 @@
 import express from 'express';
-import UsersController from '../controllers/users.controller';
+import UserController from '../controllers/user.controller';
 import CustomError from '../errors/customError';
 
 const router = express.Router();
-const usersController = new UsersController();
+const userController = new UserController();
 
-router.use(express.json());
-
-router.param('userId', usersController.handleUserIdParamMiddleware);
+router.param('userId', userController.handleUserIdParamMiddleware);
 
 router.route('/')
-    .get(usersController.getAllUsers)
-    .post(usersController.getUserBodyValidator(), usersController.createUser);
+    .get(userController.getAllUsers)
+    .post(userController.getUserBodyValidator(), userController.createUser);
 
 router.route('/:userId')
-    .get(usersController.getUser)
-    .put(usersController.getUserBodyValidator(), usersController.updateUser)
-    .delete(usersController.removeUser);
+    .get(userController.getUser)
+    .put(userController.getUserBodyValidator(), userController.updateUser)
+    .delete(userController.removeUser);
 
-router.get('/:loginSubstring/:limit', usersController.getAutoSuggestUsers);
+router.get('/:loginSubstring/:limit', userController.getAutoSuggestUsers);
```

Comment:
I think using 'loginSubstring' and 'limit' as part of url path is not good idea.
You url looks like /users/abc/10, /users/qaz/20. But if user just wants to limit user listing? You'd better use 'loginSubstring' and 'limit' as url query parameters, in this case they can be both optional.
e.g.  
/users?sub=qaz&limit=20
/users?limit=20
/users?sub=qaz
/users

Reply:
Agree, I'll make these changes